src/main/webapp/app/components/tabs/CancerRiskTabs.tsx

```diff
@@ -15,9 +15,12 @@ export interface ICancerRiskTabsProps extends StoreProps {
 }
 
 function CancerRiskTabs({ cancerRiskPath, firebaseDb, textAreaClass }: ICancerRiskTabsProps) {
-  const [cancerRisk, setCancerRisk] = useState<CancerRisk>(null);
+  const [cancerRisk, setCancerRisk] = useState<CancerRisk>();
```

Comment:
Not sure what preference is, but I usually explicitly state the initial value for useState. (without any arguments, by default its undefined).

Reply:
I generally go with `undefined` to represent no value since that's the javascript generally represents an empty value.

I prefer `useState<CancerRisk>()` over `useState<CancerRisk | null>(null)` since they basically get you the same result, but `useState<CancerRisk>()` you have to type less and there's less to read. Also since null is a special kind of empty value in JavaScript sometimes it is meaningful when you set something to null over undefined which can be confusing in some situations.